flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionITCase.java

```diff
@@ -1332,12 +1350,12 @@ private static boolean isTimestampToNumeric(LogicalType srcType, LogicalType trg
         return srcType.is(LogicalTypeFamily.TIMESTAMP) && trgType.is(LogicalTypeFamily.NUMERIC);
     }
 
-    private static <K, V> Map.Entry<K, V> entry(K k, V v) {
+    static <K, V> Map.Entry<K, V> entry(K k, V v) {
```

Comment:
Maybe we can add those 2 methods in some utility class.

Reply:
Not sure where?

Reply:
Maybe even in `CollectionUtil`, but then maybe it's too much.
Anyways it was just a quick thought.

Reply:
I like it